.github/workflows/note.yml

```diff
@@ -46,6 +54,8 @@ jobs:
     name: 'Add AOD Note'
     steps:
       - name: 'Add AOD Note'
+        if: |-
+          !contains(github.event.pull_request.labels.*.name, 'aod-breakglass')
```

Comment:
Same here, otherwise random labels can trigger this workflow too. Also, just curious can we select path for label events similar to [here](https://github.com/abcxyz/aod-template/blob/main/.github/workflows/note.yml#L22)?

Reply:
I think for consistency, we should just use this as its the same for both pull_request and pull_request_review events.